src/Cassandra/Requests/PrepareHandler.cs

```diff
@@ -34,11 +34,13 @@ internal class PrepareHandler : IPrepareHandler
 
         private readonly ISerializer _serializer;
         private readonly IInternalCluster _cluster;
+        private readonly IReprepareHandler _reprepareHandler;
 
-        public PrepareHandler(ISerializer serializer, IInternalCluster cluster)
+        public PrepareHandler(ISerializer serializer, IInternalCluster cluster, IReprepareHandler reprepareHandler)
         {
             _serializer = serializer;
             _cluster = cluster;
+            _reprepareHandler = reprepareHandler;
```

Comment:
Hmm it could but the main purpose of this was to be able to create a handler object that can re-prepare existing `PreparedStatements` without needing the dependencies to execute the first prepare request.

Reply:
sounds good.